ergotree-interpreter/src/eval/xor.rs

```diff
@@ -0,0 +1,145 @@
+use ergotree_ir::mir::value::CollKind;
+use ergotree_ir::mir::value::NativeColl;
+use ergotree_ir::mir::value::Value;
+use ergotree_ir::mir::xor::Xor;
+
+use crate::eval::env::Env;
+use crate::eval::EvalContext;
+use crate::eval::EvalError;
+use crate::eval::Evaluable;
+
+fn _helper_xor(mut x: Vec<i8>, y: Vec<i8>) -> Vec<i8> {
```

Comment:
What is the reason for the underscore prefix? I mean, there is  "silence an unused var warning" meaning employed in rustic that first comes to mind.

Reply:
Just an oversight, it has been fixed in the latest commit.